standalone/standalone.go

```diff
@@ -118,6 +122,24 @@ func Handler(ch grpcdynamic.Channel, target string, methods []*desc.MethodDescri
 	})
 }
 
+func registerExamplesHandler(mux *http.ServeMux, uiOpts *handlerOptions) {
+	var examplesBlob []byte
```

Comment:
I think this should default to `"[]"` when there are no examples. So if the `uiOpts.examples` type were `[]Example`, you could just unconditionally marshal.